release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -1165,6 +1169,14 @@ You cannot upgrade {op-system-base} 7 compute machines to {op-system-base} 8. Yo
 [id="ocp-4-10-routing-bug-fixes"]
 ==== Routing
 
+* Previously, if the cluster administrator provided a default ingress certificate that was missing the newline character for the last line, the {product-title} router would write out a corrupt PEM file for HAProxy. Now, it writes out a valid PEM file even if the input is missing a newline character. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1894431[*BZ#1894431*])
+
+* Previously, a route created where the combined name and namespace for the DNS segment greater than 63 characters long would be rejected. This expected behavior could cause problems integrating with upgraded versions of {product-title}. Now, an annotation allows non-conformant DNS hostnames. With `AllowNonDNSCompliantHostAnnotation` set to `true`, the non-conformant DNS hostname, or one longer than 63 characters, is allowed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1964112[*BZ#1964112*])
+
+* Prevoiusly, the Cluster Ingress Operator would not create wildcard DNS records for Ingress Controllers when the cluster's `ControlPlaneTopology` was set to `External`. In Hypershift clusters where the `ControlPlaneTopology` was set to `External` and the Platform is AWS, the Cluster Ingress Operator never became available. This updates limits the disabling of DNS updates when the `ControlPlaneTopology` is `External` and the platform is IBMCloud. As a result, wildcard DNS entries get created for Hypershift clusters running on AWS. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2011972[*BZ#2011972*])
+
+* Previously, the cluster ingress router was blocked from working because the Ingress Operator failed to configure a wildcard DNS record for the cluster ingress router on Azure Stack Hub IPI. With this fix, the Ingress Operator now uses the configured ARM endpoint to configure DNS on Azure Stack Hub IPI. As a result, the cluster ingress router now wrks properly. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2032566[*BZ#2032566*])
```

Comment:
```suggestion
* Previously, the cluster ingress router was blocked from working because the Ingress Operator failed to configure a wildcard DNS record for the cluster ingress router on Azure Stack Hub IPI. With this fix, the Ingress Operator now uses the configured ARM endpoint to configure DNS on Azure Stack Hub IPI. As a result, the cluster ingress router now works properly. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2032566[*BZ#2032566*])
```